apps/authoring-services/src/main/java/com/infosys/lexauthoringservices/producer/Producer.java

```diff
@@ -0,0 +1,29 @@
+package com.infosys.lexauthoringservices.producer;
+
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.infosys.lexauthoringservices.util.LexLogger;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.kafka.core.KafkaTemplate;
+import org.springframework.stereotype.Service;
+
+@Service
+public class Producer {
+
+	@Autowired
+	private LexLogger logger;
+
+	@Autowired
+	KafkaTemplate<String, String> kafkaTemplate;
+
+	public void push(String topic, Object value) {
+		ObjectMapper mapper = new ObjectMapper();
+		String message = null;
```

Comment:
message variable is unnecessary

Reply:
Removed the message unnecessary variable